bee-network/bee-autopeering/src/peer/peerlist.rs

```diff
@@ -0,0 +1,381 @@
+// Copyright 2021 IOTA Stiftung
```

Comment:
This file should be called `peer_list.rs` as per convention. Should have been called `peerlist.rs` if it was only one word but your type is called `PeerList` so it's inconsistent.

Reply:
Renamed to `lists.rs`